modules/virt-cloning-a-datavolume.adoc

```diff
@@ -0,0 +1,13 @@
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/virtual_disks/virt-cloning-a-datavolume-using-smart-cloning.adoc
+
+[id="virt-cloning-a-datavolume_{context}"]
+= Cloning a DataVolume
+
+To clone a DataVolume (DV), do the following:
+
```

Comment:
This looks like a procedure module, so you should probably have the ".Procedure" heading in front of the step.

Reply:
Done.